### ISSUE TITLE
Support src urls and alt texts on Images

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#841]
#### My Pain

I would like to use images in Panel via `pn.pane.Markdown` and/ or `pn.pane.PNG`.

Currently the Bokeh layout engine does not layout markdown with images well. See https://github.com/holoviz/panel/issues/835.

So I need to stick to `pn.pane.PNG` for images.

But the `ImageBase` class does not support parameters like the `src` url and `alt` text from the the HTML `img` tag. So I cannot provide image links or alt texts.

#### Solution

Add `src` and `alt` parameters to the `BaseImage` class.

**Comment by jbednar** [comment]
Sounds great!  This seems like an easy extension for someone to make a first PR on panel, by editing https://github.com/holoviz/panel/blob/master/panel/pane/image.py#L124 .

**Comment by philippjfr** [comment]
Hmm, not sure what you mean by `src` URL. The src URL is how the Image panes work in the first place and either contain the embedded base64 image or a URL to the image, do you mean you want to make the image clickable and link to some other URL?

**Comment by MarcSkovMadsen** [comment]
I can confirm that the link works. 

But you forgot to handle width, height and sizing_mode I believe :-) Or maybe you (@philippjfr ) made some secret AI code to always maximize images of your self :-)

-----

![image](https://user-images.githubusercontent.com/42288570/73423175-9fb02f80-432b-11ea-9d40-7d60f2034bbb.png)

```python
import panel as pn

pn.config.sizing_mode = "stretch_width"

image = pn.pane.PNG(
    "https://pbs.twimg.com/profile_images/1133352188225437696/r0KfYITe_400x400.png",
    alt="Philip",
    link_url="https://github.com/philippjfr",
    height=300,
)


app = pn.Column(image, height=300, background="blue", sizing_mode="stretch_width")
app.servable()
```
